Resources/Prototypes/_Mono/Entities/SpaceArtillery/SpaceArtillery/BallisticArtillery.yml

```
@@ -82,8 +87,71 @@
+    - trigger:
+        !type:DamageTrigger
+        damage: 600
+      behaviors:
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
+    - trigger:
+        !type:DamageTrigger
+        damage: 400
+      behaviors:
+      - !type:DoActsBehavior
+        acts: [ "Destruction" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)